v3/src/models/data/formula-fn-registry.ts

```diff
@@ -148,6 +148,24 @@ export const fnRegistry = {
     }
   },
 
+  // count(expression, filterExpression)
+  count: {
+    isAggregate: true,
+    cachedEvaluateFactory: cachedAggregateFnFactory,
+    evaluateRaw: (args: MathNode[], mathjs: any, scope: FormulaMathJsScope) => {
+      const expression = args[0]
+      const filter = args[1]
```

Comment:
You can also use:
```suggestion
      const [expression, filter] = args
```

Reply:
True, I'll update all existing functions when I add a new one.